src/pybind/mgr/dashboard/frontend/src/app/shared/components/modal/modal.component.scss

```diff
@@ -12,6 +12,7 @@
   }
 
   .modal-body {
+    min-height: 17vh;
```

Comment:
This might add weird additional space to all modals as well. Maybe we just need to increase the `z-index` or have CSS specific for component.

Reply:
Also check if this issue is for all modals.
@dnyanee1997 can you also put a **before** screenshot of this - what is it like before to compare the fix you made, thanks